test/unittests/test_zx_functionality.cpp

```diff
@@ -201,3 +200,20 @@ TEST_F(ZXDiagramTest, InitialLayout) {
     zx::fullReduce(d);
     EXPECT_TRUE(d.isIdentity());
 }
+
+TEST_F(ZXDiagramTest, RZ) {
+    qc = qc::QuantumComputation(2);
+    qc.rz(0, zx::PI / 8);
+
+    auto qcPrime = qc::QuantumComputation(2);
+    qcPrime.phase(1, zx::PI / 8);
+
```

Comment:
are these gates intentionally applied to different qubits? shouldn't there only be a single qubit in each of the circuits?

Reply:
That is certainly an oversight on my part.